lib/guardian_db.ex

```diff
@@ -17,6 +17,7 @@ defmodule GuardianDb do
 
   config = Application.get_env(:guardian_db, GuardianDb, [])
   @repo Keyword.get(config, :repo)
+  @token_types Keyword.get(config, :token_types, [])
```

Comment:
```ex
@token_types config |> Keyword.get(:token_types, []) |> Enum.map(&to_string/1)
```

Reply:
@scrogson please move out of attribute module for configurations because it will be a problem in the future.

Every time you change `token_types` values you have to recompile the dependency of `guardian_db` and no that many people are aware of that.

I suggest to create a function call `token_types` which returns exactly the same same but it doesn't need to recompile the module.

Related to https://github.com/ueberauth/guardian_db/pull/63